sycl/include/CL/sycl/handler.hpp

```diff
@@ -198,6 +198,42 @@ checkValueRange(const T &V) {
 #endif
 }
 
+template <typename TransformedArgType, int Dims, typename KernelType>
+struct RoundedRangeKernel {
```

Comment:
A minor comment: if you explicitly use 'public' and 'private' keywords here, then why using 'struct' and not 'class' ?

Reply:
Ah, yes, will change it to `class`.